lib/galaxy/tools/deps/resolvers/default_conda_mapping.yml

```diff
@@ -2,4 +2,46 @@
   to: r-base
 - from: blast+
   to: blast
+- from:
+    name: samtools
+    unversioned: true
+  to:
+    name: samtools
+    version: 1.3.1
+- from:
+    name: ucsc_tools
+    unversioned: true
+  to:
+    name: ucsc_tools
+    version: 332
```

Comment:
@mvdbeek I'm not able to find this package in our default channels, only in your Conda channel.